src/player.cpp

```diff
@@ -10008,11 +10008,19 @@ void player::mend_item( item_location&& obj, bool interactive )
     }
 }
 
-int player::item_handling_cost( const item& it, bool effects, int factor ) const {
-    int mv = std::max( 1, it.volume() * factor );
+int player::item_handling_cost( const item& it, bool effects, int factor, int qty ) const {
+
+    // If necessary create duplicate with appropriate number of charges
```

Comment:
I don't know how often this comes up, if it's more than two times or so it's be good to wrap it in a method.